xml/admin_nfsganesha.xml

```diff
@@ -9,26 +9,15 @@
  xmlns:xlink="http://www.w3.org/1999/xlink" version="5.0"
  xml:id="cha-ceph-nfsganesha">
 <!-- ============================================================== -->
- <title>&ganesha;: Export &ceph; Data via NFS</title>
- <info>
-  <dm:docmanager xmlns:dm="urn:x-suse:ns:docmanager">
-   <dm:editurl>https://github.com/SUSE/doc-ses/edit/master/xml/</dm:editurl>
-   <dm:maintainer>tbazant@suse.com</dm:maintainer>
-   <dm:status>editing</dm:status>
-   <dm:deadline/>
-   <dm:priority/>
-   <dm:translation>yes</dm:translation>
-   <dm:languages/>
-   <dm:release>SES 7</dm:release>
-  </dm:docmanager>
- </info>
+ <title>&ganesha;</title>
  <para>
   &ganesha; is an NFS server (refer to
   <link xlink:href="https://documentation.suse.com/sles/15-SP1/single-html/SLES-admin/#cha-nfs">Sharing
   File Systems with NFS</link> ) that runs in a user address space instead of
```

Comment:
```suggestion
  &ganesha; is an NFS server that runs in a user address space instead of
```

remove the link to the SLES docs? it might give the impression that NFS ganesha is configured using YaST...